### ISSUE TITLE
lstinlinline und co...

### ISSUE THREAD

**Issue by thomas-hilarius-meyer** [thomas-hilarius-meyer/LaTeX-fuer-Geisteswissenschaftler#82]
Lieber Martin, vielen Dank für Deine Korrekturen!
Unglaublich, was man so an Tippvälern produziert...

Aber sollten wir uns im Moment die Mühe machen, und jetzt einzelne lstinline-Stellen umbauen?
Ich - ein bekennender Aufschieber - dachte im Stillen, dass man das mal mit einem Suche-Ersetze-Durchgang machen wird...

**Comment by sieversMartin** [comment]
Ja, du hast Recht. Diese Korrekturen sollten global gemacht werden. Vielleicht habe ich gehofft, dass alle Autoren nochmal auf die einheitliche Kodierung aufmerksam werden... Im Moment haben wir da noch einen ziemlichen Wildwuchs.
